satpy/etc/readers/viirs_active_fires.yaml

```diff
@@ -0,0 +1,48 @@
+reader:
+    description: VIIRS Active Fires Reader
+    name: viirs_active_fires
```

Comment:
I think the name of the reader (and the name of this YAML file) should be `viirs_edr_active_fires`. Maybe `viirs_edr_fires` (@adybbroe @martin thoughts?).

Reply:
I chose`viirs_edr_fires`. Let me know if you'd prefer `viirs_edr_active_fires`.